server/server.go

```diff
@@ -188,6 +188,20 @@ func (p *proxy) newID() string {
 	return fmt.Sprintf("%x", sum)
 }
 
+// isHopByHopHeader determines whether or not the given header name represents
+// a header that is specific to a single network hop and thus should not be
+// retransmitted by a proxy.
+//
+// See: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers#hbh
+func isHopByHopHeader(name string) bool {
+	switch n := strings.ToLower(name); n {
```

Comment:
Why retain the unused variable here?

Reply:
There are no unused variables. If there were, the code wouldn't build (the Go compiler treats unused variables as build errors).

Reply:
`n` is only used in the sense that it's evaluated as a boolean. I mean to ask why not just use `strings.ToLower(name)` as the expression that's evaluated. Not a big deal either way.

Reply:
s/evaluated as a boolean/evaluated for the switch statement/

Reply:
s/evaluated as a boolean/evaluated for the switch statement/

Reply:
Ah; I understand now.

I like that a lot better; thanks for the suggestion!

Done.